web-marketplace/src/pages/GettingStarted/GettingStarted.tsx

```diff
@@ -1,4 +1,6 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
This file doesn't seem to be used anywhere and has some mock data, maybe we could just delete it?